README.md

```
@@ -1,110 +1,162 @@
-# 🛠️ GitHub Repo Template
+  # awsp — AWS profile switcher
```

markdownlint-cli2 (0.18.1)

Headings must start at the beginning of the line

(MD023, heading-start-left)